linux/install_centos7.sh

```diff
@@ -27,15 +27,21 @@ if [[ "$PGVER" =~ ^(pg96|pg10|pg11|pg12)$ ]]; then
     bash -eux step03_all_postgres.sh
 fi
 
-cp settings.env step04_all_omero.sh setup_omero_db.sh ~omero-server
-
 bash -eux step01_centos7_ice_venv.sh
 
+if [ "$(getent passwd omero-server)" ]; then
+    cp settings.env step04_all_omero.sh setup_omero_db.sh ~omero-server
+fi
+
+
+
 OMEROVER=$OMEROVER ICEVER=$ICEVER bash -eux step04_all_omero_install.sh
 
-su - omero-server -c " bash -eux step04_all_omero.sh"
+if [ "$(getent passwd omero-server)" ]; then
```

Comment:
Am I correct that the goal of this extra logic is to have `install_centos7.sh` as the single entry-point but with support different installation paths?

Is it possible to exaplin why these steps are skipped?